### ISSUE TITLE
Add ability to run audits with different selection strategy

### ISSUE THREAD

**Issue by pipermerriam** [ethereum/glados#38]
(low priority, only useful once the network is actually mostly working and most audits are successful)

depends on: https://github.com/pipermerriam/glados/issues/37

Lets call the strategy laid out in #37 the `latest` strategy, as it focuses on auditing the newest stuff.

We want two more strategies, and to modify `glados-audit` in any way necessary to allow us to run multiple audit processes concurrently (which might require database locking of some sort).

- A `random` strategy, that simply randomly audits things.
- A `missing` strategy that looks for failed audits and checks whether the data is still missing.

**Comment by perama-v** [comment]
Keeping track of the set changes from PRs:
- `latest`
- `random`
- `failed` (prev `missing`)
